array_methods_lab.js

```diff
@@ -148,8 +156,9 @@ Output: 384
 \n----------------------------------------\n\n`);
 
 let qar9 = [1, 2, 3, 4]
-let ans9 = qar9.reduce((i, j) => i * j * 2, 1)
-console.log(ans9);
+const productOfAllDoubled = (arr) => arr.reduce((i, j) => i * j * 2, 1)
+// let ans9 = qar9.reduce((i, j) => i * j * 2, 1)
+console.log(productOfAllDoubled(qar9));
```

Comment:
Doesn't quite work. We have to iterate through the array and double each value then find the sum.